src/Api/SecretsManager/Controllers/AccessPoliciesController.cs

```diff
@@ -12,16 +18,34 @@ namespace Bit.Api.SecretsManager.Controllers;
 public class AccessPoliciesController : Controller
 {
     private readonly IAccessPolicyRepository _accessPolicyRepository;
+    private readonly IServiceAccountRepository _serviceAccountRepository;
     private readonly ICreateAccessPoliciesCommand _createAccessPoliciesCommand;
+    private readonly ICurrentContext _currentContext;
     private readonly IDeleteAccessPolicyCommand _deleteAccessPolicyCommand;
+    private readonly IGroupRepository _groupRepository;
+    private readonly IOrganizationUserRepository _organizationUserRepository;
+    private readonly IProjectRepository _projectRepository;
     private readonly IUpdateAccessPolicyCommand _updateAccessPolicyCommand;
+    private readonly IUserService _userService;
 
     public AccessPoliciesController(
+        IUserService userService,
+        ICurrentContext currentContext,
         IAccessPolicyRepository accessPolicyRepository,
+        IServiceAccountRepository serviceAccountRepository,
+        IGroupRepository groupRepository,
+        IProjectRepository projectRepository,
+        IOrganizationUserRepository organizationUserRepository,
         ICreateAccessPoliciesCommand createAccessPoliciesCommand,
         IDeleteAccessPolicyCommand deleteAccessPolicyCommand,
         IUpdateAccessPolicyCommand updateAccessPolicyCommand)
     {
+        _userService = userService;
+        _serviceAccountRepository = serviceAccountRepository;
+        _projectRepository = projectRepository;
+        _currentContext = currentContext;
+        _groupRepository = groupRepository;
+        _organizationUserRepository = organizationUserRepository;
```

Comment:
Let's group repositories and services.

```suggestion
        _userService = userService;
        _currentContext = currentContext;
        _serviceAccountRepository = serviceAccountRepository;
        _projectRepository = projectRepository;
        _groupRepository = groupRepository;
        _organizationUserRepository = organizationUserRepository;
```

Reply:
Resolved in https://github.com/bitwarden/server/pull/2628/commits/79bce819b50d75b5ce021b4b4dfad21dc9c97cd8